.github/CONTRIBUTING.md

```
@@ -177,4 +177,33 @@ WA-->>Dev: Workflow completion result
+ *
+ * @example
+ * const data = { foo: "bar", count: 1 }
+ * console.log(toMarkdownList(data))
+ * // - foo: bar
+ * // - count: 1
+ */
+function toMarkdownList(obj: Record<string, any>): string {
+  return Object.entries(obj)
+    .map(([key, value]) => `- ${key}: ${value}`)
+    .join('\n')
+}
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)